src/app/(proper_react)/redesign/(authenticated)/user/dashboard/page.tsx

```diff
@@ -37,7 +41,12 @@ export default async function DashboardPage() {
   const scanResult = await getLatestOnerepScan(profileId);
   const scanResultItems = scanResult?.onerep_scan_results?.data ?? [];
   const breaches = await getUserBreaches({ user: session.user });
-  const summary = dashboardSummary(scanResultItems, breaches);
+  const subBreaches = await getSubscriberBreaches(session.user);
+  console.log(JSON.stringify(subBreaches));
+  const summary = dashboardSummary(scanResultItems, subBreaches);
+  console.log(JSON.stringify(summary));
+  const guidedBreaches = guidedExperienceBreaches(subBreaches);
+  console.log(JSON.stringify(guidedBreaches));
```

Comment:
```suggestion
  const subBreaches = await getSubscriberBreaches(session.user);
  const summary = dashboardSummary(scanResultItems, subBreaches);
  const guidedBreaches = guidedExperienceBreaches(subBreaches);
```

Reply:
changes in `page.tsx` are just to show @codemist what the object changes looks like, it will be cleaned up in: https://github.com/mozilla/blurts-server/pull/3299